src/commands/broadcast-song/index.js

```diff
@@ -0,0 +1,72 @@
+const multiLocationSpecificSong = require("../multi-location-specific-song");
+const multiLocationSpotifySong = require("../multi-location-spotify-song");
+const multiLocationWildcardSong = require("../multi-location-wildcard-song");
+
+const displaySongChoice = async function(
+  bot,
+  updateMessage,
+  message,
+  { artist, name },
+  isDryRun,
+  locationName
+) {
+  const dryRunPrefix = isDryRun
+    ? "if this hadn't been a dry run :beach_with_umbrella: I'd have "
+    : "";
+
+  await updateMessage(
+    `Thanks <@${message.user}>, ` +
+      dryRunPrefix +
+      `queued "${name}" by ${artist} :man-with-bunny-ears-partying:,` +
+      ` in ${locationName}:round_pushpin:`
+  );
+};
```

Comment:
Can this make use of the `displaySongChoice` util in `src/commands/utils/display-song-choice.js`?

Reply:
This was copy & paste from somewhere else, but I can do a little more refactoring here.